app/domain/authentication/authn_jwt/validate_status.rb

```diff
@@ -0,0 +1,114 @@
+module Authentication
+  module AuthnJwt
+
+    ValidateStatus = CommandClass.new(
+      dependencies: {
+        create_signing_key: Authentication::AuthnJwt::CreateSigningKeyInterface.new,
+        fetch_issuer_value: Authentication::AuthnJwt::FetchIssuerValue.new,
+        validate_uri_parameters: Authentication::AuthnJwt::ValidateUriBasedParameters.new,
+        fetch_identity_from_token: Authentication::AuthnJwt::IdentityFromDecodedTokenProvider,
+        validate_webservice_is_whitelisted: ::Authentication::Security::ValidateWebserviceIsWhitelisted.new,
+        validate_role_can_access_webservice: ::Authentication::Security::ValidateRoleCanAccessWebservice.new,
+        validate_webservice_exists: ::Authentication::Security::ValidateWebserviceExists.new,
+        enabled_authenticators: Authentication::InstalledAuthenticators.enabled_authenticators_str(ENV),
+        validate_account_exists: ::Authentication::Security::ValidateAccountExists.new,
+        logger: Rails.logger
+      },
+      inputs: %i[authenticator_status_input]
+    ) do
+      extend(Forwardable)
+      def_delegators(:@authenticator_status_input, :authenticator_name, :account,
+                     :username, :webservice, :status_webservice)
```

Comment:
i suppose `webservice` is the` authn-jwt/servic-id` webservice, how do we have this info?